automatic/pdfcreator/tools/chocolateyinstall.ahk

```diff
@@ -0,0 +1,68 @@
+DetectHiddenWindows, on
+__Welcome:
+; Welcome window
+WinWait, Setup - PDFCreator, Welcome to the PDFCreator Setup Wizard
+WinHide, Setup - PDFCreator, Welcome to the PDFCreator Setup Wizard
+; Next button
+ControlClick, &Next, Setup - PDFCreator, Welcome to the PDFCreator Setup Wizard
```

Comment:
What if there is another setup window ? Not very likely but you should probably get a hwnd of the single one and work with `ahk_id %hwnd%` afterwards. Even better would be for AHK to actually start the installer and get its hwnd.

Not very high probability of errors tho.

Reply:
Looking more into this its not possible to do it with ahk_id since windows probably change but they all probably belong to same PID which can be used to filter out unwanted windows. 

Not terribly important tho.